pkg/onchain/website/uploader.go

```diff
@@ -30,7 +32,8 @@ func PrepareForUpload(url string, wallet *wallet.Wallet) (string, error) {
 }
 
 type UploadWebsiteParam struct {
-	Data string `json:"data"`
+	Data        string `json:"data"`
```

Comment:
This feal weird : don't you need an offset to identify the chunk ?

Reply:
Not yet, it will be handled by this issue : 
[https://github.com/massalabs/thyra/issues/81](https://github.com/massalabs/thyra/issues/81)